kie-wb-common-react/boxed-expression-editor/src/__tests__/components/FunctionExpression/FunctionExpression.test.tsx

```diff
@@ -270,4 +288,91 @@ describe("FunctionExpression tests", () => {
       ).toBeTruthy();
     });
   });
+
+  describe("JAVA Function Kind", () => {
+    test("should show, by default, an entry with a context table, containing two entries: class and method", () => {
+      const { container } = render(
+        usingTestingBoxedExpressionI18nContext(
+          <FunctionExpression logicType={LogicType.Function} functionKind={FunctionKind.Java} formalParameters={[]} />
+        ).wrapper
+      );
+
+      expect(container.querySelector(".function-expression table tbody td.data-cell")).toBeVisible();
+      expect(container.querySelector(".function-expression table tbody td.data-cell .context-expression")).toBeTruthy();
+      expect(
+        container.querySelectorAll(
+          ".function-expression table tbody td.data-cell .context-expression .context-entry-info-cell"
+        )
+      ).toHaveLength(2);
+      expect(
+        container.querySelectorAll(
+          ".function-expression table tbody td.data-cell .context-expression .context-entry-info-cell"
+        )[0]
+      ).toContainHTML("class");
+      expect(
+        container.querySelectorAll(
+          ".function-expression table tbody td.data-cell .context-expression .context-entry-info-cell"
+        )[1]
+      ).toContainHTML("method");
+    });
+
+    test("should show an entry corresponding to the passed class and method values", () => {
+      const classValue = "class value";
+      const methodValue = "method value";
+
+      const { container } = render(
+        usingTestingBoxedExpressionI18nContext(
+          <FunctionExpression
+            logicType={LogicType.Function}
+            functionKind={FunctionKind.Java}
+            formalParameters={[]}
+            class={classValue}
+            method={methodValue}
+          />
+        ).wrapper
+      );
+
+      expect(
+        container.querySelectorAll(
+          ".function-expression table tbody td.data-cell .context-expression .context-entry-expression-cell"
+        )
+      ).toHaveLength(2);
+      expect(
+        container.querySelectorAll(
+          ".function-expression table tbody td.data-cell .context-expression .context-entry-expression-cell"
+        )[0]
+      ).toContainHTML(classValue);
+      expect(
+        container.querySelectorAll(
+          ".function-expression table tbody td.data-cell .context-expression .context-entry-expression-cell"
+        )[1]
+      ).toContainHTML(methodValue);
+    });
+  });
```

Comment:
I askled my team mates, if this can't be simplified to sth. like:
```
      expect(
        container.querySelectorAll(
          ".function-expression table tbody td.data-cell .context-expression .context-entry-expression-cell"
        )
      ).extracting("HTML")
      .containAll(classValue, methodValue);
```